app/policies/standup_meeting_policy.rb

```diff
@@ -12,12 +10,10 @@ def update?
   private
 
   def matching_user?
-    standup_meeting.user_id == user.id
+    record.user_id == user.id
   end
 
-  class Scope < Scope
-    def resolve
-      user.standup_meetings
-    end
+  def group_member?
+    record.users.where(id: user.id).present?
```

Comment:
should just do `find_by` because we are looking for a single record. Technically doesn't make much of a difference just more clear.

Reply:
Since it's a boolean check for one record, I really like to reach for `#exists?` [Rails guide for it](https://guides.rubyonrails.org/active_record_querying.html#existence-of-objects)

But then you can express this as `record.users.exists?(user.id)`